app.tests.ts

```diff
@@ -1,6 +1,9 @@
 import { red } from "ansicolor";
 import { getManifestFileName, main } from "./app";
 import { version } from "./package.json";
+import { cpSync, mkdtempSync, rmdirSync, writeFileSync } from "fs";
+import { join, resolve } from "path";
+import { tmpdir } from "os";
```

Comment:
Those (many) imports usually hint that we could instead use a test double (mock or stub).